src/annotator/annotation-sync.js

```diff
@@ -40,25 +40,32 @@ export default class AnnotationSync {
      */
     this.cache = {};
 
-    this._on = options.on;
-    this._emit = options.emit;
+    this.destroyed = false;
 
     // Relay events from the sidebar to the rest of the annotator.
     this.bridge.on('deleteAnnotation', (body, callback) => {
+      if (this.destroyed) {
+        callback(new Error('AnnotationSync has been destroyed')); // this causes the bridge channel to be destroyed
```

Comment:
Destroying the bridge channel will affect other consumers of the `Bridge` instance, which is unexpected. I think it would be better to just return no result here. `FrameSyncService` in the sidebar doesn't actually use the result of this call, so perhaps we could just make this RPC handler return void in all cases.

Reply:
I have implemented your suggestion of returning no results in all cases.